_extras/checkout.md

```diff
@@ -102,44 +94,37 @@ Please note that:
+    our lessons are already full to bursting. If you are a Software Carpentry instructor and are
+    submitting an exercise, we will accept contributions to our [new Software Carpentry Python lesson]({{ site.swc_github}}/python-novice-gapminder/).
 
-6.  We do *not* want exercises or other material that introduce new concepts:
-    our lessons are already full to bursting.
-
-7.  If you are writing an exercise,
-    we strongly prefer contributions to the Data Carpentry lessons right now (since that material is much younger)
-    or to our fledgling "Python for complete novices" lesson.
-
-8.  If you would rather do a diagram, we are happy to take submissions to any of our lessons.
+6.  We are happy to accept contributions of new diagrams for any of our lessons.
     Please use SVG, since vector formats are easier to manipulate than raster formats.
 
-9.  We also appreciate improvements to our lessons' learning objectives.
-    In particular, we would like checked against the actual lesson content,
-    and would also like them labelled according to the levels of Bloom's Taxonomy.
+7.  We also appreciate improvements to our lessons' learning objectives.
+    In particular, we would like our current learning objectives to be checked against the actual lesson content.
 
 The lessons you may submit to are:
 
-1.  [Data Carpentry lessons]({{ site.dc_site }}/lessons/):
+1.  [For Data Carpentry instructors]({{ site.dc_site }}/lessons/):
     we would prefer contributions to these right now.
     *   R-ecology: <{{ site.dc_github }}/R-ecology/>
     *   Python-ecology: <{{ site.dc_github }}/python-ecology/>
     *   SQL-ecology: <{{ site.dc_github }}/sql-ecology/>
     *   OpenRefine-ecology: <{{ site.dc_github }}/OpenRefine-ecology/>
-2.  [New Software Carpentry Python lesson]({{ site.swc_github}}/python-novice-gapminder/):
-    This lesson is an experiment to see if we can start collaboration much earlier than we have in the past.
-    If you want to add to this, please submit an exercise that uses the Gapminder data set (in the "data" directory).
-    Please don't worry too much about formatting, and please only submit the Markdown files (not generated HTML) in pull requests.
-3.  [Stable Software Carpentry lessons]({{ site.swc_site }}/lessons/):
-    we are currently accepting solutions to existing exercises, diagrams, and improvements to learning objectives.
+2.  [For Software Carpentry instructors]({{ site.swc_site }}/lessons/): 
+    we are currently accepting solutions to existing exercises, diagrams, and improvements to learning objectives
+    for any lesson. We are also accepting new exercises using the Gapminder data set (in the "data" directory) for 
+    our [new Software Carpentry Python lesson]({{ site.swc_github}}/python-novice-gapminder/).
+    Please don't worry too much about formatting, and please only submit the Markdown files 
+    (not generated HTML) in pull requests.
+
```

Comment:
Add a third point saying that we accept diagrams and new lesson objectives for all lessons?